### ISSUE TITLE
About how to specifying frame rate when recording videos

### ISSUE THREAD

**Issue by FantomasTsang** [Genymobile/scrcpy#5243]
Hello, may I specify a recording frame rate when recording videos? I tried adding -- max-fps 60, but the recorded video didn't have the expected 60 frames per second.
    If frame rate can be specified, please tell me how to operate it。
    Thanks

**Comment by rom1v** [comment]
> The frame rate is intrinsically variable: a new frame is produced only when the screen content changes. For example, if you play a fullscreen video at 24fps on your device, you should not get more than 24 frames per second in scrcpy.

https://github.com/Genymobile/scrcpy/blob/master/doc/video.md#frame-rate

**Comment by FantomasTsang** [comment]
> > 帧速率本质上是可变的：只有当屏幕内容发生变化时才会产生新帧。例如，如果您在设备上以 24fps 的速度播放全屏视频，则 scrcpy 中的帧速率不能超过每秒 24 帧。
> 
> https://github.com/Genymobile/scrcpy/blob/master/doc/video.md#frame-rate

yes,I read this document before and thought there may be another way to record at a specified frame rate.
anyway,thank you.

**Comment by rom1v** [comment]
MediaCodec does not provide an option to give a constant frame rate. One possibility is to record to a file, then resample using ffmpeg.

**Comment by FantomasTsang** [comment]
> MediaCodec does not provide an option to produce a constant frame rate. One possibility is to record to a file, then resample using ffmpeg.

That could be a way.  i am thinking if i could run a app on the screen, which keep display a small area video at high frame rate. then i use scrcpy to record the whole screen, may be i can get my idea frame rate video?
